code/game/objects/items/weapons/cigs.dm

```diff
@@ -452,6 +452,7 @@ LIGHTERS ARE IN LIGHTERS.DM
 	name = "Holo-Cigar"
 	desc = "A sleek electronic cigar imported straight from Sol. You feel badass merely glimpsing it..."
 	icon_state = "holocigaroff"
+	clothing_traits = list(TRAIT_BADASS)
```

Comment:
this trait should only be applied when the holocigar is on

Reply:
Removed this in favour of converting this in a seperate PR.